runtime/doc/dev_style.txt

```diff
@@ -1009,37 +989,6 @@ expr;`. >
     return(result);  // return is not a function!
 
 
-Preprocessor Directives ~
-
-The hash mark that starts a preprocessor directive should always be at the
-beginning of the line.
-
-Even when preprocessor directives are within the body of indented code, the
-directives should start at the beginning of the line.
-
-Nested directives should add one spaces after the hash mark for each level of
-indentation. 
```

Comment:
does uncrustify take care of this "nested indent" stuff? that's pretty nice if so, but if not we should continue to document it since I see it missed in some PRs

Reply:
It does, and it's very good at it in general. Please link to me if you encounter any places it's formatting incorrectly so I can forward it to the Uncrustify team.

Reply:
oh, I meant humans miss this sometimes. crusty is 🤌